src/config/shared.js

```diff
@@ -32,3 +32,38 @@ export function extractBucketMetadata(transformedData) {
   etc.bucketSortOrder = sorter;
   return etc;
 }
+
+export const facets = {
+  activeBuckets: {
+    isMulti: true,
+    defaultValue: [],
+  },
+  aggregationLevel: {
+    defaultValue: 'build_id',
+  },
+  channel: {
+    defaultValue: 'nightly',
+  },
+  os: {
+    defaultValue: 'Windows',
+  },
+  process: {
+    defaultValue: 'content',
+  },
+  product: {
+    defaultValue: 'firefoxDesktop',
+  },
+  proportionMetricType: {
+    defaultValue: 'proportions',
+  },
+  reference: {
+    defaultValue: '',
+  },
+  timeHorizon: {
+    defaultValue: 'MONTH',
+  },
+  visiblePercentiles: {
+    isMulti: true,
+    defaultValue: [95, 75, 50, 25, 5],
+  },
+};
```

Comment:
Another item to possibly add here, or maybe it comes with a future code change, is `versions`, as in, the number of versions to query data for, defaulting to 3 if not sent to the API.

Reply:
Let's make sure to file these as follow-on issues.

Reply:
@robhudson Would you mind opening issues for this and the thing about firefox.desktop. I don't completely understand what you mean. I'll open an issue about OS defaults.